### PR TITLE
Temp update on production deploy config to test CI

### PR DESCRIPTION
**Pull request by limsohee1002**
### Description
Temporary update on production deploys config to test production deployment on the staging site. 
I wanted to test that all the environment variables worked correctly, so I added some console.log to double-check. 

We ultimately need to check if 
- noindex tag should not be present on production data
- local data should be used to generate the site 
## status
- Blocked by #65

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -115,5 +115,7 @@ workflows:
           filters:
             branches:
               only: production
-          s3-uri: s3://marketing-website-assets-905418149376
-          cloudfront-distribution-id: E2KP9J5JTG1DGX
+          s3-uri: s3://marketing-website-assets-905418149376-stg
```
Comment (shivananda-sahu) [inline .circleci/config.yml:118]: if we do need to test just the staging deployment and skip the prod deployment - would just comment out the prod deployment temporarily

File: .circleci/config.yml
Comment (shivananda-sahu) [inline .circleci/config.yml:1]: Targeted this to `feat/add-staging` to make reviews slightly easier

Comment (limsohee1002) [reply .circleci/config.yml:1]: This is a great idea—very smart! I’ll update this PR once #65 is approved. Then, before merging, we can test here and proceed with merging #65 afterward.

File: .circleci/config.yml (hunk 1 above)
Comment (shivananda-sahu) [reply .circleci/config.yml:118]: @limsohee1002  thoughts on this one? lets either
- only deploy to staging and comment out the prod job
- deploy to both staging & prod from the main branch - and have the console.log statements tell us the output of the steps

Comment (limsohee1002) [reply .circleci/config.yml:118]: I think we should test deploy both staging & prod from the main branch to staging aws site. 
The build config is different between staging and production, so I want to make sure to add console.log statements to check the build config is working correctly if that is ok with you 🙏

**Pull request closed without merging**